packages/graph/hash_graph/lib/graph/Cargo.toml

```diff
@@ -23,8 +23,14 @@ uuid = { version = "1.1.2", features = ["v4", "serde"] }
 utoipa = { git = "https://github.com/juhaku/utoipa", rev = "031073f", features = ["uuid"] }
 
 [dev-dependencies]
+tokio = { version = "1.18.2", features = ["rt", "rt-multi-thread"] }
 
 [build-dependencies]
 
 [features]
 clap = ["dep:clap"]
+
+[[test]]
+test = false
```

Comment:
What's the reasoning for this?

Reply:
This avoids running the tests by default. 

> Disable running the integration tests by default (will run in CI, can be run by passing --test integration)

Reply:
I agree, hard to find as this can be specified for any target, not only for tests. For reference: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#configuring-a-target.